homeassistant/components/fully_kiosk/services.py

```diff
@@ -81,3 +97,19 @@ async def async_start_app(call: ServiceCall) -> None:
                 )
             ),
         )
+
+    hass.services.async_register(
+        DOMAIN,
+        SERVICE_SET_CONFIG,
+        async_set_config,
+        schema=vol.Schema(
+            vol.All(
+                {
+                    vol.Required(ATTR_DEVICE_ID): cv.ensure_list,
+                    vol.Required(ATTR_CONFIG_TYPE): vol.In(["string", "bool"]),
```

Comment:
If you are going to go with a config_type to determine the type of `ATTR_VALUE`, you need to use `cv.key_value_schemas`